install

```diff
@@ -31,6 +32,10 @@ if uname | egrep -qe "Linux"; then
     fi
 elif uname | egrep -qe "MINGW64"; then
     ENV_FILE=$WIN64_ENV
+elif uname | egrep -qe "Darwin"; then
+    if uname -m | egrep -qe "arm64"; then
+        ENV_FILE=$MACOS_ARM64_ENV
+    fi
```

Comment:
This change is not related with the PR. Please remove it.

Reply:
Undid changes to env